lib/orb/memory.ex

```diff
@@ -3,7 +3,7 @@ defmodule Orb.Memory do
   Work with memory: load, store, declare pages & initial data.
   """
 
-  defstruct name: "", min: 0, exported_name: "memory"
+  defstruct name: "", min: 0, exported_name: "memory", is_imported?: false
```

Comment:
There‘s a bug from me where we aren’t using the `exported_name` field in the wat output. So I should fix that.

I think generally it makes sense if you can export xor import, not both. So I’ll have a think about how best to represent that…

Reply:
Suggested resolution? Or needs some thinking?

Reply:
Sorry work has been super busy, I’ll resolve this soon!